qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -16,6 +16,21 @@
     no Windows..i386
     no WinXP Win2000 Win2003 WinVista
     sub_test_wait_time = 0
+    variants:
+        - op_on:
+            dump_mem = yes
+            merge_mem = yes
+            prealloc_mem = yes
+            only one
+            only no_policy
+            only backend_ram
+        - op_off:
+            dump_mem = no
+            merge_mem = no
+            prealloc_mem = no
+            only one
+            only no_policy
+            only backend_ram
```

Comment:
These 3 lines are same option for op_on, need to reorganize.

Reply:
Thanks, updated.